config/routes/support.rb

```diff
@@ -10,6 +10,7 @@
       end
       resource :onboarding, only: %i[new create]
     end
+    resources :providers_contact_details, only: %i[edit update]
```

Comment:
Just thinking, this could probably be nested under `resource :providers`? but just the non pluralised version `resource :contact_details, only: %i[edit update], path: 'contact-details'`

then you would get:

`/providers/122/contact-details/edit`

It's not a blocker but we still need the dash paths to be consistent with the other support paths